### ISSUE TITLE
resources: allow to register shorthand resource callback

### ISSUE THREAD

**Issue by x1B** [LaxarJS/laxar-patterns#67]
Currently, the `onUpdateReplace` callback has to be wrapped in the options-object.
Since onUpdateReplace is *the* use case for the resource handler, especially when AngularJS is not used, it should be allowed to pass *just* the function (without any wrapper).
This also makes the API more similar to the action-handler API.

**Comment by x1B** [comment]
We should do the same for `flags`

**Comment by x1B** [comment]
Implemented on `master` (`v2.0.0`) and `master-1.x` (`v1.3.0`).

### Upgrade Information

The resource- and flag-handlers now support a more convenient syntax for registering handler methods:

```js
import { resources, flags } from 'laxar-patterns'; // ... or equivalent AMD-style import

var rh = resources.handlerFor( $scope );

// equivalent when `f` is a function:
rh.registerResource( 'topic, { onUpdateReplace: f } );
rh.registerResource( 'topic', f );

// equivalent when `f` is a function:
rh.registerResourceFromFeature( 'featurePath', { onUpdateReplace: f } );
rh.registerResourceFromFeature( 'featurePath', f );


var fh = flags.handlerFor( $scope );

// equivalent when `f` is a function:
fh.registerFlag( 'topic, { onChange: f } );
fh.registerFlag 'topic', f );

// equivalent when `f` is a function:
rh.registerFlagFromFeature( 'featurePath', { onChange: f } );
rh.registerFlagFromFeature( 'featurePath', f );
```
